### PR TITLE
chore(ember): update to latest version of ember - 3.16

### PR DESCRIPTION
**Pull request by svsalvador**
BREAKING CHANGE: this update may break your app.

### CODE REVIEW

File: tests/dummy/config/ember-cli-update.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (adrigzr) [inline tests/dummy/config/ember-cli-update.json]: What is this file?

File: package.json [hunk 2]
```diff
@@ -30,49 +30,78 @@
     "url": "https://github.com/BBVAEngineering/ember-cli-markdown-codefences/issues"
   },
   "scripts": {
-    "lint": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
-    "test": "mocha node-tests/*test.js "
+    "lint": "npm run lint:js && npm run lint:hbs",
+    "lint:hbs": "ember-template-lint .",
+    "test": "mocha node-tests/*test.js",
```
Comment (adrigzr) [inline package.json]: No coverage in this repo?

Comment (svsalvador) [reply package.json]: I can add coverage but I think that there is no point in adding coverage to this kind of tasks.

File: tests/dummy/config/ember-cli-update.json (hunk 1 above)
Comment (svsalvador) [reply tests/dummy/config/ember-cli-update.json]: Removed

**Pull request merged**